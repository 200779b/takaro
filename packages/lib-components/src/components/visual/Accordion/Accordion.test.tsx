import { Accordion } from '.';
import { render } from 'test-utils';

it('Should render <Accordion />', () => {
  const { container } = render(<Accordion title="Accordion title" />);
  expect(container).toMatchSnapshot();
});
