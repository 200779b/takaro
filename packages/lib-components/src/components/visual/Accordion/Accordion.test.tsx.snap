// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render <Accordion /> 1`] = `
.c0 {
  margin: 0.5rem 0;
  border-radius: 0.5rem;
  height: 100%;
  background-color: transparent;
  will-change: height;
  -webkit-transition: 0.2s height ease-in-out;
  transition: 0.2s height ease-in-out;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 1rem 1rem 0 1rem;
  min-height: 40px;
  cursor: pointer;
}

.c1 h3 {
  color: #030917;
  font-weight: 700;
  font-size: 1rem;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 18px;
  height: 18px;
  border-radius: 50%;
  background-color: white;
  margin-right: 1rem;
}

.c2 svg {
  fill: #664DE5;
}

<div>
  <div
    class="c0"
    color="primary"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
        color="primary"
      >
        <svg
          fill="currentColor"
          height="15"
          stroke="currentColor"
          stroke-width="0"
          t="1551322312294"
          version="1.1"
          viewBox="0 0 1024 1024"
          width="15"
          xmlns="http://www.w3.org/2000/svg"
        >
          <defs />
          <path
            d="M474 152m8 0l60 0q8 0 8 8l0 704q0 8-8 8l-60 0q-8 0-8-8l0-704q0-8 8-8Z"
          />
          <path
            d="M168 474m8 0l672 0q8 0 8 8l0 60q0 8-8 8l-672 0q-8 0-8-8l0-60q0-8 8-8Z"
          />
        </svg>
      </div>
      <h3>
        Accordion title
      </h3>
    </div>
  </div>
</div>
`;
